_studio/shared/include/libmfx_core_vaapi.h

```diff
@@ -44,6 +44,42 @@
 #include "mfx_vpp_interface.h"
 #endif
 
+//helper struct, it is help convert linux GUIDs to VAProfile and VAEntrypoint
+struct VaGuidMapper
+{
+    VAProfile profile;
+    VAEntrypoint entrypoint;
+
+    VaGuidMapper() = default;
```

Comment:
well, you don't need it, this constructor actually does nothing.
maybe you wanted this one?
vaGuidMapper(VAProfile prof, VAEntrypoint entryp) = default;

Reply:
Sorry, I didn't understand you the first time. default use with default constructor.